config.py

```diff
@@ -20,13 +21,25 @@ class Config:
     LOG_DATE_FORMAT = os.getenv('LOG_DATE_FORMAT', '%Y-%m-%dT%H:%M%s')
     LOG_JSON_INDENT = os.getenv('LOG_JSON_INDENT')
     LOG_LEVEL_PIKA = os.getenv('LOG_LEVEL_PIKA', 'ERROR')
+    SFTP_HOST = os.getenv('SFTP_HOST')
+    SFTP_PORT = os.getenv('SFTP_PORT')
+    SFTP_USERNAME = os.getenv('SFTP_USERNAME')
+    SFTP_KEY_FILENAME = os.getenv('SFTP_KEY_FILENAME')
+    SFTP_PASSPHRASE = os.getenv('SFTP_PASSPHRASE')
+    SFTP_DIRECTORY = os.getenv('SFTP_DIRECTORY')
 
 
 class DevConfig(Config):
     RABBIT_HOST = os.getenv('RABBIT_HOST', 'localhost')
     RABBIT_PORT = os.getenv('RABBIT_PORT', '6672')
     RABBIT_USERNAME = os.getenv('RABBIT_USERNAME', 'guest')
     RABBIT_PASSWORD = os.getenv('RABBIT_PASSWORD', 'guest')
+    SFTP_HOST = os.getenv('SFTP_HOST', 'localhost')
+    SFTP_PORT = os.getenv('SFTP_PORT', '122')
+    SFTP_USERNAME = os.getenv('SFTP_USERNAME', 'centos')
+    SFTP_KEY_FILENAME = os.getenv('SFTP_KEY_FILENAME', 'dummy_keys/dummy_rsa')
+    SFTP_PASSPHRASE = os.getenv('SFTP_PASSPHRASE', 'secret')
+    SFTP_DIRECTORY = os.getenv('SFTP_DIRECTORY', 'Documents/sftp/print_service')
```

Comment:
Needs to be able to write to different SFTP directories for different suppliers.